jmclient/jmclient/wallet_utils.py

```diff
@@ -1275,6 +1349,8 @@ def wallet_tool_main(wallet_root_path):
         return wallet_signmessage(wallet_service, options.hd_path, args[2])
     elif method == "freeze":
         return wallet_freezeutxo(wallet_service, options.mixdepth)
+    elif method == "gettimelockaddress":
+        return wallet_gettimelockaddress(wallet_service, args[2])
```

Comment:
Probably want to handle a missing argument with a meaningful error message here.

Reply:
Will do